pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -75,7 +75,7 @@ static const char* s_anisotropic_filtering_entries[] = {QT_TRANSLATE_NOOP("Graph
 	QT_TRANSLATE_NOOP("GraphicsSettingsWidget", "8x"), QT_TRANSLATE_NOOP("GraphicsSettingsWidget", "16x"), nullptr};
 static const char* s_anisotropic_filtering_values[] = {"0", "2", "4", "8", "16", nullptr};
 
-static constexpr int DEFAULT_INTERLACE_MODE = 7;
+static constexpr int DEFAULT_INTERLACE_MODE = 0;
```

Comment:
Shouldn't this be different? As far as I know interlacing 0 is none interlacing.
```suggestion
static constexpr int DEFAULT_INTERLACE_MODE = 8;
```

Reply:
Ok never mind, You changed the value. My bad.

Was looking at the GameIndexMD:
`        {Automatic Off, WeaveTFF, WeaveBFF, BobTFF, BobBFF, BlendTFF, BlendBFF, AdaptiveTFF, AdaptiveBFF}`